Add newline parsing for symbol map name

DrDeano opened the pull request:
The symbol name can have spaces.
Changed the parseName to use the newline parsing.
Also used str.len
Closes #199

SamTebbs33 commented on src/kernel/panic.zig:
```diff
@@ -217,6 +217,28 @@ fn parseNonWhitespace(ptr: [*]const u8, end: *const u8) PanicError![*]const u8 {
+///
+/// Arguments:
+///     IN ptr: [*]const u8 - The address at which to start looking.
+///     IN end: *const u8 - The end address at which to start looking. A whitespace character must
```
Do you mean a new-line character rather than a whitespace character?

DrDeano replied:
yep, missed one :P

DrDeano committed:
Add newline parsing

The symbol name can have spaces.
Changed the parseName to use the newline parsing.
Also used str.len

Missed whitespace => newline


Add a nothing to run release safe tests
